### PR TITLE
[webserver] Make BE webserver handle static files

### PR DESCRIPTION
**Pull request by acelyc111**
Make BE webserver handle static files, e.g. css, js, ico, then we can make
BE website more pretty.

### CODE REVIEW

File: be/src/common/config.h [hunk 1]
```diff
@@ -536,6 +536,8 @@ namespace config {
     // Whether to continue to start be when load tablet from header failed.
     CONF_Bool(ignore_load_tablet_failure, "false");
 
+    // Directory where BE's website files placed.
+    CONF_String(www_path, "${DORIS_HOME}/www");
```
Comment (morningman) [inline be/src/common/config.h:540]: Is it necessary to be a config? I think it can be hard coded?

Comment (acelyc111) [reply be/src/common/config.h]: Seems not necessary, I'll remove it.

**Review by morningman — APPROVED** [review]
LGTM

**Comment by acelyc111** [comment]
A step for https://github.com/apache/incubator-doris/issues/3911

**Pull request merged**